tests/simulator/refinement/test_2d_overlaps.py

```
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)